src/zeit/cms/tagging/interfaces.py

```diff
@@ -124,3 +124,13 @@ def __call__(ressort=None):
         if no ressort is given)"""
 
     headlines = zope.interface.Attribute('List of headline keywords')
+
+
+class ITopicpages(zope.interface.Interface):
+    """Utility to retrieve (paginated) information about topic pages."""
+
+    def get_topics(start=0, rows=25):
+        """Returns lists of dicts with keys ``id`` and ``title``.
+
```

Comment:
Erhält man eine oder mehrere Listen? Aktuell erhält man ein `Result`. Die Dicts können potentiell auch mehr keys enthalten, vielleicht `with at least keys id and title`?